front/app/containers/ProjectsShowPage/shared/poll/index.tsx

```diff
@@ -63,10 +57,10 @@ interface Props extends InputProps, DataProps {}
 
 const disabledMessages: { [key in PollDisabledReason] } = {
   project_inactive: messages.pollDisabledProjectInactive,
-  not_active: messages.pollDisabledNotActiveUser,
-  not_verified: messages.pollDisabledNotVerified,
-  missing_data: messages.pollDisabledNotActiveUser,
-  not_signed_in: messages.pollDisabledMaybeNotPermitted,
+  not_active: undefined,
+  not_verified: undefined,
+  missing_data: undefined,
+  not_signed_in: undefined,
   not_in_group: globalMessages.notInGroup,
```

Comment:
Wasn't quite sure what to do here - we need to include all keys in PollDisabledReason, but we won't ever shown messages for these reasons since they are fixable with authentication :thinking: I can't remove them from PollDisabledReason either since they come from PermissionsDisabledReason. Not sure if you have any suggestions for a better approach here? :)

Reply:
Hmm maybe you can type `disabledMessages` as `Partial<Record<PollDisabledReason, MessageDescriptor>>`